modules/users/client/controllers/signup.client.controller.js

```diff
@@ -82,6 +75,14 @@
      */
     function activate() {
 
+      if (! Authentication.user && vm.invitationCode) {
+        validateInvitationCode(
+          appSettings.invitations.key, // inviteKey
+          new Date(), // today
+          vm.invitationCode.toLowerCase() // code
+        );
+      }
```

Comment:
I changed how this works recently; make sure to fetch the latest master get latest changes.

You can simply do this here and it'll handle the rest:
```js
if (vm.invitationCode) {
  validateInvitationCode();
}
```

Also no need to check for `Authentication.user` as it's done here already:

https://github.com/Trustroots/trustroots/blob/master/modules/users/client/controllers/authentication.client.controller.js#L11-L12